backend/src/test/java/gov/cdc/usds/simplereport/service/UploadServiceTest.java

```diff
@@ -48,132 +44,139 @@ void setupData() {
   @Test
   void testRowWithValue() {
     String value = this._service.getRow(Map.of("key1", "value1"), "key1", true);
-    assertEquals("value1", value);
+    assertThat(value).isEqualTo("value1");
   }
 
   @Test
   void testRowWithEmptyValue() {
     String value = this._service.getRow(Map.of("key1", ""), "key1", false);
-    assertEquals("", value);
+    assertThat(value).isEqualTo("");
```

Comment:
Just a heads up that Sonar is flagging this as a code smell; it wants you to use `isEmpty()` instead

Reply:
Honestly I kinda prefer to check that its putting out exactly `""`, isEmpty() is probably that exact same but I like to be explicit

Reply:
Fair enough! Just wanted to flag it